spec/requests/api/v1/merchants/revenue_request_spec.rb

```diff
@@ -0,0 +1,60 @@
+describe "Revenue Business Intelligence Methods" do
```

Comment:
Nice Tests, like the setup. We could probably refine this a bit more with our factories